### PR TITLE
chore(website): set cache-control for static assets

### PR DESCRIPTION
**Pull request by nschonni**
## Motivation

Mentioned as one of the "issues" in the Lighthouse reports. These get unique names, so there shouldn't be a risk in a long cache life

### Have you read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/main/CONTRIBUTING.md#pull-requests)?

(Write your answer here.)

## Test Plan

(Write your test plan here. If you changed any code, please provide us with clear instructions on how you verified your changes work. Bonus points for screenshots and videos!)

## Related PRs

(If this PR adds or changes functionality, please take some time to update the docs at https://github.com/facebook/docusaurus, and link to your PR here.)

### CODE REVIEW

**Comment by nschonni** [comment]
Hmm, went down. Wonder if it is conflicting with https://github.com/facebook/docusaurus/blob/main/website/static/_headers

**Comment by Josh-Cena** [comment]
Could be. Not familiar with Netlify setup, but seems that's where headers go.

File: website/static/_headers [hunk 1]
```diff
@@ -3,16 +3,26 @@
 # They are safe for immutable caching, as filename change when content change
 
 /assets/*
-  cache-control: immutable
+  Cache-Control: public
+  Cache-Control: max-age=365000000
```
Comment (Josh-Cena) [inline website/static/_headers:7]: These are not the static assets though :D So not sure you did what you originally wanted?

**Comment by nschonni** [comment]
I'm seeing the headers in DevTools now, but it's still complaining

**Comment by Josh-Cena** [comment]
Not sure if it did anything. @nschonni can you point me to where the lighthouse was complaining originally?

**Comment by nschonni** [comment]
You can see the message on any of the PRs right now like https://storage.googleapis.com/lighthouse-infrastructure.appspot.com/reports/1642336500065-12272.report.html

**Comment by slorber** [comment]
That looks fine thanks, didn't know immutable support was that bad 😅 

Don't understand why Lighthouse still report these URLs though...

**Pull request merged**